README.md

```diff
@@ -50,6 +59,42 @@ located in `fdroidcl`'s config directory. This will be
 You can run `fdroidcl defaults` to create the config with the default
 settings.
 
+### Automation
+
+Example script to automate install & update of apps:
+
+    #!/bin/bash
+    set -e
+
+    # Example list of apps to install/update.
+    #
+    # You can create a list like this from existing manually installed apps
+    # by doing fdroidcl search -i, then manually removing automatically
+    # installed and preinstalled apps
+    fdroid_pkgs=(
+        at.bitfire.davdroid
+        com.nutomic.syncthingandroid
+        org.dmfs.tasks # gui name: OpenTasks
+        org.fdroid.fdroid
+        org.smssecure.smssecure
+    )
+
+    declare -A installed updated
+    fdroidcl update
+    for p in $(fdroidcl search -i| grep -o "^\S\+"); do
+        installed[$p]=true
+    done
+    for p in $(fdroidcl search -u| grep -o "^\S\+"); do
+        updated[$p]=false
+    done
+    for p in ${fdroid_pkgs[@]}; do
+        ${installed[$p]:-false} || fdroidcl install $p
+    done
+    for p in ${!installed[@]}; do
+        ${updated[$p]:-true} || fdroidcl upgrade $p
+    done
+
+
```

Comment:
This is a nice little script, but it's too large to include in the README. Worse, it will be hard to tell if it works or not - or even if the bash syntax is broken or not. I also don't want to maintain bash as part of this project.

I think we should either:

* Include a similar feature in `fdroidcl`, like `install -u` that would either install or upgrade the apps it is given, skipping the ones already installed and up to date
* Keep this script in an actual shell script file somewhere else, like a `fdroidcl-contrib` repo

Inclined towards the former. A patch would be welcome, but I can also work on that eventually.